docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -161,6 +171,7 @@ Preparing For a Release
    In addition, review the link-check step of the documentation builder to ensure that all the errors (the links with "[broken]") are expected.
 
 #. Update the staging documentation.
+   (For a documentation release ``${VERSION}`` be the base release version in this step).
```

Comment:
This is horrible. I'm going to address this by having a tool that adjusts things automatically.